README.md

```diff
@@ -16,3 +16,7 @@
 ## License
 
 This SDK was created under the [MIT License](LICENSE).
+
+## Contributing
+
+1. Run ./setup-hooks.sh (sets up commit hooks on your local repository)
```

Comment:
can we execute this shell file in npm prepare script?

Reply:
if we want to run this before publish for which prepare command is usually used  then 

instead of using this shell script we can directly add  test and build command 

"scripts": {
        "prepare": "vitest --run && tsc && vite build"
         }

Reply:
Yeah let's add it on the prepare script.